### PR TITLE
Hello-Microverse

### PR DESCRIPTION
**Pull request by zunairkhan811**
Adding Linters, index.html, styles.css, test.md, .gitignore files in this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,97 @@
-# Hello-Microverse-Repository
\ No newline at end of file
+# HTML & CSS3 Course
+
+If you are not familiar with linters, read [root level README](../README.md).
```
Comment (BregornOriginal) [inline README.md]: - [x] Remember that the objective of this lesson and from now on is to follow the next [README TEMPLATE OF MICROVERSE](https://github.com/microverseinc/readme-template). Kindly use that template and complete it with the information about this protect 💯 💯 💯

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+node_modules
+Test.md
```
Comment (BregornOriginal) [inline .gitignore:2]: ✅ Good work ignoring node_modules and Test.md files 🥇

File: README.md [hunk 3]
```diff
@@ -1 +1,97 @@
...
+
```
Comment (BregornOriginal) [inline README.md:2]: - [x] Remember that when we create a Pull Request, the person that it's going to review your proyect needs to know on the first view the changes that you're doing on that PR. Kindly add a description of the changes that you are making on this PR, you can use the requirements of this activity for help 💯

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
## Hi @zunairkhan811  

## Good job so far!
## There are some issues that you still need to work on to go to the next project but you are almost there!


one-step-closer-one-more


## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good Pull Request title
✅ Good use of GitHub Flow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## You can also consider:

[OPTIONAL] **From now on remember to use meaningful commits. Instead of ` Added` you have to say `Add`


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (zunairkhan811) [reply README.md:2]: So should I make changes to these as above you have written that these are optional?

**Review by Yerimah — APPROVED** [review]
# Status: Approved 🥇🥇 

Hi @zunairkhan811 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f474b3767725a594c47376373302f67697068792e676966

## Highlights
✅ Linters are passing
✅ Good commit history
✅ Good Pull Request title
✅ Good use of GitHub Flow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**